examples/sample_config.yml

```
@@ -6,7 +6,28 @@ appenders:
         filters:
             - kind: threshold
               level: info
+    file:
+        kind: file
+        path: "log/log.log"
+        encoder:
+            pattern: "[{d(%Y-%m-%dT%H:%M:%S%.6f)} {h({l}):<5.5} {M}] {m}{n}"
+    rollingfile:
+        kind: rolling_file
+        path: "log/log2.log"
+        encoder:
+            pattern: "[{d(%Y-%m-%dT%H:%M:%S%.6f)} {h({l}):<5.5} {M}] {m}{n}"
+        policy:
+            trigger: 
+                kind: time
+                limit: 1 minute 
+            roller: 
+                kind: fixed_window
+                pattern: "log/old-log-{}-.log" 
```

Probably don't need the extra dash
```suggestion
                pattern: "log/old-log-{}.log" 
```